test/unit/spec/MfaVerify_spec.js

```diff
@@ -690,6 +690,22 @@ function (Okta,
             expect(test.form.errorMessage()).toBe('Your answer doesn\'t match our records. Please try again.');
           });
         });
+        itp('shows errors if verify button is clicked and answer is empty', function () {
+          return setupSecurityQuestion()
+          .then(function (test) {
+            $.ajax.calls.reset();
+            test.setNextResponse(resSuccess);
```

Comment:
we probably don't need to set a next response, since we won't make a request.

Reply:
Done